frontend/src/components/schools/Schools.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { connect } from 'react-redux'
+import { Button } from '@material-ui/core'
+import Paper from '@material-ui/core/Paper'
+import AddCircleIcon from '@material-ui/icons/AddCircle'
+
+import * as actions from '../../store/actions'
+import SchoolCard from './SchoolCard'
+import SchoolForm from './SchoolForm'
+
+const Wrapper = styled.div`
+  width: 70%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 20px;
+`
+
+const HeadingContainer = styled.div`
+  margin-bottom: 1rem;
+`
+
+const SchoolsContainer = styled.ul`
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+
+  & .school-row {
+    margin-bottom: 1rem;
+  }
+`
+
+const Schools = ({ schools: { schools }, getAllSchools }) => {
+  const [schoolFormVisible, setSchoolFormVisible] = useState(false)
+  const [schoolFormEditMode, setSchoolFormEditMode] = useState(false)
+  const [editingSchool, setEditingSchool] = useState(false)
```

Comment:
Change Request (:bowing_man: ):

Since we `editingSchool` will be a school object once user clicks edit button, it's a bit confusing to set a default Boolean value.

So, can we make the initial value as null please?